### PR TITLE
Added Asana documentation

### PR DESCRIPTION
**Pull request by kostaspap**
## Background

Why are you making this change? Please provide a reference to any related issues and PRs

Added documentation for onboarding Asana destination

## Changes

- Updated docs with instructions to use Asana destination

## Testing

- This PR only includes documentation changes. These changes are visible through this PR. 
I have followed this instructions to successfully add an Asana destination in my test account

### CODE REVIEW

File: docs/gitbook/destinations/alert-setup/asana.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Asana
+
+This page will walk you through configuring [Asana](https://asana.com) as a Destination for your Panther alerts.
+
+The Asana destination requires an `Access Token` and one or more `Project GID`. When an alert is forwarded to an Asana destination, a new task is created in the the specified project(s).
+
+![](../../.gitbook/assets/add_asana_destination.png)
+
+The Asana Destination is integrated through use of an Asana user account. To perform the Asana integration, first select a user to be the reporter of the issue. If possible, a service account should be created specifically for this purpose in order to ensure continuity.
```
Comment (3nvi) [inline docs/gitbook/destinations/alert-setup/asana.md]: > The Asana Destination is integrated through use of an Asana user account.

Could rephrase to `The Asana Destination requires the use of an Asana account.`

Comment (3nvi) [reply docs/gitbook/destinations/alert-setup/asana.md]: > To perform the Asana integration, first select a user to be the reporter of the issue

Could rephrase to `To add an Asana destination, first select a user as the reporter of the issue`

Comment (3nvi) [reply docs/gitbook/destinations/alert-setup/asana.md]: > for this purpose

Needs a comma after the word `purpose`

File: docs/gitbook/destinations/alert-setup/asana.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Asana
+
+This page will walk you through configuring [Asana](https://asana.com) as a Destination for your Panther alerts.
```
Comment (3nvi) [inline docs/gitbook/destinations/alert-setup/asana.md]: `Destination` should be lowercased

File: docs/gitbook/destinations/alert-setup/asana.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+The Asana destination requires an `Access Token` and one or more `Project GID`. When an alert is forwarded to an Asana destination, a new task is created in the the specified project(s).
+
+![](../../.gitbook/assets/add_asana_destination.png)
+
+The Asana Destination is integrated through use of an Asana user account. To perform the Asana integration, first select a user to be the reporter of the issue. If possible, a service account should be created specifically for this purpose in order to ensure continuity.
+
+A `Project GID` is a unique identifier for the project where Panther will be creating new issues. The easiest way to retrieve it, is from a project URL. You can retrieve selecting `Copy Project Link` on a project.
```
Comment (3nvi) [inline docs/gitbook/destinations/alert-setup/asana.md]: > You can retrieve selecting `Copy Project Link` on a project.

Would change to: 

```
The easiest way to retrieve a project URL, is by clicking `Copy Project Link` on a project
```

File: docs/gitbook/destinations/alert-setup/asana.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+![](../../.gitbook/assets/destination_asana_copy_project_link.png)
+
+The copied link will be in a form like
```
Comment (3nvi) [inline docs/gitbook/destinations/alert-setup/asana.md]: Could rephrase to: `The copied link should look like`

File: docs/gitbook/destinations/alert-setup/asana.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+`https://app.asana.com/0/1160306557000000/board`. The Project GID is the 16-digit number seein in the URL.
```
Comment (3nvi) [inline docs/gitbook/destinations/alert-setup/asana.md]: typo in `seen`

File: docs/gitbook/destinations/alert-setup/asana.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+The `Access Token` is the token of the user that will be creating the issues. This can be acquired by going to [https://app.asana.com/0/developer-console](https://app.asana.com/0/developer-console) while logged in as the user and clicking on `New Access Token`.
+
+![](../../.gitbook/assets/destination_asana_new_access_token.png)
+
+After creating the token, you will have an opportunity to copy it. As Asana warns this token should be treated just as sensitively as a password and you will not be able to access it again in the future.
```
Comment (3nvi) [inline docs/gitbook/destinations/alert-setup/asana.md]: Missing a comma: `As Asana warns, this token...`

File: docs/gitbook/destinations/alert-setup/asana.md (hunk 1 above)
Comment (kostaspap) [reply docs/gitbook/destinations/alert-setup/asana.md]: Rephrased.

File: docs/gitbook/destinations/alert-setup/asana.md (hunk 2 above)
Comment (kostaspap) [reply docs/gitbook/destinations/alert-setup/asana.md]: Done

File: docs/gitbook/destinations/alert-setup/asana.md (hunk 3 above)
Comment (kostaspap) [reply docs/gitbook/destinations/alert-setup/asana.md]: Rephrased.

File: docs/gitbook/destinations/alert-setup/asana.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# Asana
+
+This page will walk you through configuring [Asana](https://asana.com) as a destination for your Panther alerts.
+
+The Asana destination requires an `Access Token` and one or more `Project GID`. When an alert is forwarded to an Asana destination, a new task is created in the the specified project(s).
+
+![](../../.gitbook/assets/add_asana_destination.png)
```
Comment (rleighton) [inline docs/gitbook/destinations/alert-setup/asana.md]: Missing description between [] ... similar for all image links below.

Comment (kostaspap) [reply docs/gitbook/destinations/alert-setup/asana.md]: I followed the style we use in the rest of destination documentation... E.g. https://raw.githubusercontent.com/panther-labs/panther/master/docs/gitbook/destinations/alert-setup/slack.md or https://raw.githubusercontent.com/panther-labs/panther/master/docs/gitbook/destinations/alert-setup/pagerduty.md

Comment (rleighton) [reply docs/gitbook/destinations/alert-setup/asana.md]: Austin asked me to do the same on the docs I was doing (even tho I was following what you did w/out the labels)... I guess we are inconsistent. I think it is generally a good idea tho.

Comment (kostaspap) [reply docs/gitbook/destinations/alert-setup/asana.md]: Agreed. Updating....

**Comment by jacknagz** [comment]
@kostaspap make sure to add this page to the [SUMMARY](https://github.com/panther-labs/panther/blob/asanadest/docs/gitbook/SUMMARY.md), or else it won't show in the navigation.

File: docs/gitbook/destinations/alert-setup/asana.md (hunk 7 above)
Comment (kostaspap) [reply docs/gitbook/destinations/alert-setup/asana.md]: Added

**Comment by kostaspap** [comment]
> @kostaspap make sure to add this page to the [SUMMARY](https://github.com/panther-labs/panther/blob/asanadest/docs/gitbook/SUMMARY.md), or else it won't show in the navigation.

Added!

**Pull request merged**